sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientException.java

```diff
@@ -193,13 +194,13 @@ void setError(CosmosError cosmosError) {
     }
 
     /**
-     * Gets the recommended time interval after which the client can retry failed
+     * Gets the recommended time duration after which the client can retry failed
      * requests
      *
-     * @return the recommended time interval after which the client can retry failed
+     * @return the recommended time duration after which the client can retry failed
      * requests.
      */
-    public long getRetryAfterInMilliseconds() {
+    public Duration getRetryAfterDuration() {
```

Comment:
Duration is not used as a suffix in ConnectionPolicy#setRequestTimeout() and getter.

any reason we are using it here?

what are your thought on the method naming?

Reply:
The reason I added Duration here is because `getRetryAfter` seems very general. It was not clear to me that it returns duration to retry after. So I added that. I can change it to `getRetryAfter` if you think that is much better.

Reply:
ok makes sense, `getRetryAfterDuration()` looks good